### DISCUSSION TITLE
how can I add more attribute in User Type in nextAuth.js

### DISCUSSION THREAD

**Discussion by kaushal9808** [nextauthjs/next-auth#4417] [Help]
### Question 💬

I wanted to add some more parameter but only name, email, image is only getting in session but it is available in token.

### How to reproduce ☕️

you can login and get the user object from session it showing only default properties

### Contributing 🙌🏽

No, I am afraid I cannot help regarding this

**Comment by relez** [comment]
Hi there, I am trying to figure out how to add custom properties to the session object. Any idea?

Thanks!

**Reply by rjgux** [reply]
I'm also trying to figure this out!
